tgui/packages/tgui/styles/components/Button.scss

```
@@ -134,3 +136,19 @@ $bg-map: colors.$bg-map !default;
 .Button--selected {
   @include button-color($color-selected);
 }
+
+.Button--verticalAlign--top {
+	align-items: flex-start;
+  }
+  
+  .Button--verticalAlign--middle {
+	align-items: center;
+  }
+  
+  .Button--verticalAlign--bottom {
+	align-items: flex-end;
+  }
+  
+  .Button__content--fluid {
+	flex: 1 0 auto;
+  }
```

How is it so hard to format CSS properly?

```suggestion
.Button--verticalAlign--top {
	align-items: flex-start;
}
  
.Button--verticalAlign--middle {
	align-items: center;
}
  
.Button--verticalAlign--bottom {
	align-items: flex-end;
}
  
.Button__content--fluid {
	flex: 1 0 auto;
}
```